src/pybind/mgr/restful/api/request.py

```diff
@@ -77,6 +77,11 @@ def post(self, **kwargs):
         """
         Pass through method to create any request
         """
+        if isinstance(request.json, list):
+            command_list = []
+            for command in request.json:
```

Comment:
This for doesn't do much, you can just pass request.json direcly to the submit_request method when you detect a `list`.